src/semeio/forward_models/scripts/design_kw.py

```diff
@@ -44,7 +46,7 @@ def create_parser():
         default="WARNING",
         type=logging.getLevelName,
     )
-
+    parser.add_argument("--validate", required=False, type=bool, default=True)
```

Comment:
I thought default is `False`?

Reply:
It is false in `ForwardModelStepPlugin`'s default_mapping, but I can have it default False here too :)

Reply:
So why we need both? :)

Reply:
I think it is so that someone running it directly as a script won't have to specify it every time. 
We do the same for `--log_level`

Reply:
Clarified!